src/modules/task/task.controller.ts

```diff
@@ -109,6 +109,15 @@ export class TaskController {
 
     return {
       message: 'Tarefas atualizadas',
+      data: {
+        childrenId: updateTaskDto.childrenId,
```

Comment:
```suggestion
        childrenId: updateResult.childrenId,
```

Na linha "100", pegue o resultado da atualização e o use para montar esses dados de resposta. Isso porque o servidor pode ter alterado alguma informação que veio do _client_ (formatado, concatenado, etc.).